sdk/tools/gen_baseaddress.py

```diff
@@ -92,7 +92,7 @@
     'bootvid.dll',
     'framebuf.dll',
     'framebuf_new.dll',
-    'freeldr_pe.dll',
+    'freeldr_pe.exe',
```

Comment:
Looking at the base addresses for .exes do not make sense. You can remove this line.

Reply:
More exactly freeldr "goes away" once the rest of the OS is loaded.

Reply:
exactly why this is the 'EXCLUDE' list

Reply:
My point was that you don't also add notepad.exe, calc.exe, etc... into this list, so there is no reason to specifically add freeldr_pe.exe .